tldraw/tldraw
# fix(shapes): resolve shape fills for ellipse and rectangle

This PR fixes the shape fills for ellipses and rectangles.

Before: 
![image](https://user-images.githubusercontent.com/23072548/136700670-bc72a3f7-20fe-44ce-8ae4-2f0435757a39.png)

After:
![image](https://user-images.githubusercontent.com/23072548/136700686-4e9e8343-4f89-4b5c-b81e-4e69a90cb234.png)

### Change type


### Test plan

1. Create an ellipse or rectangle shape.
2. Apply a fill style.
3. Verify the fill renders correctly.


### Release notes

- Fixed an issue where shape fills for ellipses and rectangles were not rendering correctly in draw style.
